app.rambox.ramboxce.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+ <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should be contributed upstream.

Reply:
And since upstream uses just `rambox.desktop`, then this should be `rambox.metainfo.xml` to match, then use `rename-appdata-file`.